### PR TITLE
Feat/clippy 38 v2

### PR DESCRIPTION
**Pull request by patr1ck001**
Error handling with toast messages

### CODE REVIEW

**Review by MihaelsGit — COMMENTED** [review]
Should we support multiple toasts or just replace the toast that is currently displayed?
@IvanJuricic @patr1ck001

File: src/components/FileUpload.js [hunk 1]
```diff
@@ -6,32 +6,66 @@ import FileDropzone from "./FileDropzone";
...
 import "../styles/Link.css";
+import "../styles/error.css";
+
+import { useNavigate } from "react-router-dom";
+import toast, { Toaster } from 'react-hot-toast';
 
-import { Link } from "react-router-dom";
 export default function FileUpload({ setFileId }) {
-  const [pdfFile, setPdf] = useState(null);
+  const [pdfFile, setPdf] = useState(null); // Initialize with a value !=NULL pdfFile to render correctly the first time. Can be set to null after the user dop the file
```
Comment (IvanJuricic) [inline src/components/FileUpload.js]: Please no comments in source code

**Review by IvanJuricic — CHANGES_REQUESTED** [review]

File: src/components/FileDropzone.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ function FileDropzone({ setPDFFile }) {
       setPDFFile(null);
       setDropzoneText(dropzoneError);
     }
+    setPDFFile(selectedFile);
     // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (AvendoOT) [inline src/components/FileDropzone.js:31]: Why eslint disable?

**Review by AvendoOT — DISMISSED** [review]

File: src/components/FileUpload.js (hunk 1 above)
Comment (patr1ck001) [reply src/components/FileUpload.js]: Sorry that was supposed to be only for me then I forgot it there

File: src/components/FileDropzone.js (hunk 2 above)
Comment (patr1ck001) [reply src/components/FileDropzone.js:31]: I do not know, in fact, I haven't disabled it myself.

File: src/components/FileUpload.js (hunk 1 above)
Comment (IvanJuricic) [reply src/components/FileUpload.js]: No worries, just commit everything without comments

**Review by IvanJuricic — APPROVED** [review]

**Pull request merged**